contrib/go-redis/redis.v8/redis.go

```diff
@@ -143,6 +145,8 @@ func (ddh *datadogHook) BeforeProcessPipeline(ctx context.Context, cmds []redis.
 		tracer.ResourceName(raw[:strings.IndexByte(raw, ' ')]),
 		tracer.Tag("redis.args_length", strconv.Itoa(length)),
 		tracer.Tag("redis.pipeline_length", strconv.Itoa(len(cmds))),
+		tracer.Tag(ext.Component, "go-redis"),
```

Comment:
redis.v7 / redis.v8 integrations have the same component name. I'm not sure, but it's worth looking up if it should instead be:
```suggestion
		tracer.Tag(ext.Component, "go-redis.v8"),
```

Reply:
And .v7 in the other part as well

Reply:
Done